### DISCUSSION TITLE
IATI

### DISCUSSION THREAD

**Discussion by erikvaningen** [openkfw/TruBudget#1299] [Q&A]
Dear TruBudget community,

Some of the principles/concepts of your community are similar to [IATI](https://iatistandard.org/en/). How do you the two relate to each other?

Best,
Erik

**Comment by jzakotnik** [comment]
Dear Erik,

yes, you are right. Do you refer to the data standard or the data they are already hosting on various projects?

One potential vision is that using a distributed ledger, more transparent information could be relayed to a public blockchain in real-time. This could be later aggregated and processed by transparency organisations (like IATI), but it could also be available as Open Data. For example I was impressed by a [prototype](https://nrc-cnrc.explorecatena.com/en) some years ago in Canada, where all tenders were published in a public ethereum blockckain. Using APIs, this could also be done in TruBudget, so we're experimenting with this technically.

One question remains, which data can be published in public (as open data) and for which data there might be regulatory requirements (e.g. that it has to be deleted after x years), where this could not be done.

Hope this helps :-) Jure
